src/main/java/com/playdata/eungae/appointment/controller/AppointmentApiController.java

```diff
@@ -46,4 +49,10 @@ public List<LocalTime> findTime(
 
 		return appointmentService.createAppointmentPossibleTime(appointmentDate, appointmentDayOfWeek, doctorSeq, hospitalSeq);
 	}
+
+	@PatchMapping("/my/appointments")
+	@ResponseStatus(HttpStatus.ACCEPTED)
```

Comment:
HTTP status code accepted 의 의미는 다음과 같아요 

> The HyperText Transfer Protocol (HTTP) 202 Accepted response status code indicates that the request has been accepted for processing, but the processing has not been completed; in fact, processing may not have started yet. The request might or might not eventually be acted upon, as it might be disallowed when processing actually takes place.
> 202 is non-committal, meaning that there is no way for the HTTP to later send an asynchronous response indicating the outcome of processing the request. It is intended for cases where another process or server handles the request, or for batch processing.

요청이 받아들여 졌다는 응답을 서버에서 클라이언트에게 바로 보내주고 이후 서버에서는 계속 처리하고 있다는 의미 입니다
지금 delete 관련 동작은 delete가 이뤄지고 응답을 보내주게 되는데 이 의미랑은 맞지 않는거 같아요
다른 response status 값을 고민 해보시겠어요?

Reply:
한번 찾아보고 반영하겠습니다!